prov/efa/src/efa_fabric.c

```diff
@@ -907,25 +944,19 @@ int efa_fabric(struct fi_fabric_attr *attr, struct fid_fabric **fabric_fid,
 	int ret = 0;
 
 	/*
-	 * Enable rdma-core fork support and huge page support. We want call
-	 * this only when the EFA provider is selected. It is safe to call this
-	 * function again if multiple EFA fabrics are opened or if the fabric
-	 * is closed and opened again.
-	 *
-	 * TODO: allow users to disable this once the fork() to check ptrace
-	 * permissions is removed.
+	 * setting RDMAV_HUGEPAGES_SAFE alone will not impact
+	 * application peformance. It is only effective
```

Comment:
"It is only effective" is ambiguous.  Maybe "rdma-core only checks this value when ..." or something similar?

Reply:
Thank you! fixed in new revision.